Sources/SpeziLLMOpenAI/FunctionCalling/LLMFunctionParameterWrapper+ArrayTypes.swift

```diff
@@ -32,20 +32,25 @@ extension _LLMFunctionParameterWrapper where T: AnyArray, T.Element: BinaryInteg
         maxItems: Int? = nil,
         uniqueItems: Bool? = nil
     ) {
-        self.init(schema: .init(
-            type: .array,
-            description: String(description),
-            items: .init(
-                type: .integer,
-                const: const.map { String($0) },
-                multipleOf: multipleOf,
-                minimum: minimum.map { Double($0) },
-                maximum: maximum.map { Double($0) }
-            ),
-            minItems: minItems,
-            maxItems: maxItems,
-            uniqueItems: uniqueItems
-        ))
+        do {
+            try self.init(schema: .init(unvalidatedValue: [
+                "type": "array",
+                "description": String(description),
+                "items": [
+                    "type": "integer",
+                    "const": const.map { String($0) } as Any?,
+                    "multipleOf": multipleOf as Any?,
+                    "minimum": minimum.map { Double($0) } as Any?,
+                    "maximum": maximum.map { Double($0) } as Any?
+                ].compactMapValues { $0 },
+                "minItems": minItems as Any?,
+                "maxItems": maxItems as Any?,
+                "uniqueItems": uniqueItems as Any?
+            ].compactMapValues { $0 }))
+        } catch {
+            logger.error("LLMFunctionParameterWrapper+ArrayTypes")
```

Comment:
Probably a more descriptive error message would be beneficial here.
In general, I think we shouldn't just ignore this error! (how does the current procedure play out down the road when OpenAI function calling parameters are decoded and injected into the Swift DSL?)
The downside is a potentially throwing init, which we definitely don't want here.

Reply:
Once we've worked out https://github.com/StanfordSpezi/SpeziLLM/pull/64#discussion_r1894626727, we can probably apply the same solution here, I hope.